install/operator/pkg/cmd/internal/run/run.go

```diff
@@ -23,13 +23,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"github.com/syndesisio/syndesis/install/operator/pkg/openshift"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
-
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
-
 	"github.com/syndesisio/syndesis/install/operator/pkg/apis"
 	"github.com/syndesisio/syndesis/install/operator/pkg/controller"
+	"github.com/syndesisio/syndesis/install/operator/pkg/openshift"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
```

Comment:
This should be updated to `sigs.k8s.io/controller-runtime/pkg/log` according to https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#import-updates.

Would be good to check the other imports listed in the migration doc

Reply:
Checked and included, thanks for pointing this out!

Reply:
no problem at all :-)